decidim-comments/app/types/decidim/comments/author_type.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+module Decidim
+  module Comments
+    # This type represents an author who owns a resource
+    AuthorType = GraphQL::ObjectType.define do
+      name "Author"
+      description "An author"
+
+      field :name, !types.String
```

Comment:
It's not necessary right now. It's different with `comments` because we are rendering a list and I need something as a `key` prop.